Extensions/people_notifier.js

```diff
@@ -311,13 +311,16 @@ XKit.extensions.people_notifier = new Object({
 		} else {
 
 			var current_ms = new Date().getTime();
-
+			var text_color = "rgba(255, 255, 255, 0.5)";
+			if ($('body').hasClass('dashboard_useraction_following')) {
```

Comment:
If we just wanted to check if we were on the `/following` page, I think it would be better to add a field for that in `XKit.interface.where()`.